engine/src/main/java/org/terasology/logic/characters/KinematicCharacterMover.java

```diff
@@ -459,8 +459,9 @@ private boolean moveHorizontal(Vector3f horizMove, CharacterCollider collider, V
         }
         boolean horizontalHit = false;
         Vector3f normalizedDir = horizMove.normalize();
-        if(!normalizedDir.isFinite())
+        if(!normalizedDir.isFinite()) {
```

Comment:
Oh boy, we still had bare `if`s? Yikes :-)

Trivial: One more space needed here